server/routes/profile_test.go

```diff
@@ -29,14 +29,9 @@ import (
 	jsonpatch "github.com/evanphx/json-patch"
 )
 
-func init() {
-	ctx := context.Background()
-	dsclient.Init(ctx)
-}
-
 func fillDatastore() {
 	ctx := context.Background()
-	dsClient := dsclient.DsClient()
+	dsClient := services.Locator.DsClient()
```

Comment:
Did you change the contents of the dsclient? I'm assuming you've moved it into services, but you haven't pushed it

Reply:
Yeah, I did it in the previous PR, though I forgot to change this test unfortunately

Reply:
Ah okay